package.json

```diff
@@ -41,7 +41,6 @@
     "applicationinsights": "^0.19.0",
     "chalk": "^1.1.3",
     "commander": "^2.9.0",
-    "node-status": "^1.0.0",
```

Comment:
Are you referring to issue #9, which is about a hang when a nonexistent file is specified? I've fixed that one on line 38.